server/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ api:
     - INITIAL_ADMIN_CODE=initialadmincode
   links:
    - mongodb
+  labels:
+   - "io.kontena.container.skip_logs=1"
```

Comment:
yep, I just put that in to prevent my mac from melting down on exponential logging :)